source/playgrounds.txt

```diff
@@ -30,12 +30,40 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+- If you enabled split-view, |vsce| splits your Playground window to
+  display the results of your Playground.
+
+  - If the results can be displayed in |json| format, |vsce| labels the
+    right-side pane **Playground Results.json**.
```

Comment:
It never has an extension at the end, it always `Playground Result` (and Result not Result**s**).

Reply:
The difference between displaying json and not json document:
- json documents have syntax highlighting
- json documents have the `Edit Document` code lens